core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeAstBuilder.scala

```diff
@@ -64,7 +65,14 @@ class SnowflakeAstBuilder extends SnowflakeParserBaseVisitor[ir.LogicalPlan] wit
     case c => c.accept(dmlBuilder)
   }
 
+  override def visitOtherCommand(ctx: OtherCommandContext): ir.LogicalPlan = {
+    ctx.accept(commandBuilder)
+  }
+
   override def visitSnowSqlCommand(ctx: SnowSqlCommandContext): ir.UnresolvedCommand = {
-    ir.UnresolvedCommand(ctx.getText)
+    ctx match {
+      case c if ctx.SQLCOMMAND() != null => ir.UnresolvedCommand(ctx.getText)
+      case _ => null
```

Comment:
why can't it be unresolved command in all cases?

Reply:
Reverted